src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,68 @@
+import { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+
+const centuries = ['16', '17', '18', '19', '20'];
+
 export const PeopleFilters = () => {
+  const [query, setQuery] = useState('');
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const currentSex = searchParams.get('sex') ?? '';
+  const currentCenturies = searchParams.getAll('centuries') ?? [];
+
+  const handleQueryChange = (queryValue: string) => {
+    setQuery(queryValue);
+
+    if (queryValue === '') {
+      setSearchParams(getSearchWith(searchParams, { query: null }));
+    } else {
+      setSearchParams(getSearchWith(searchParams, { query: queryValue }));
+    }
+  };
+
+  const handleToggleCentury = (century: string): string[] => {
+    return currentCenturies.includes(century)
+      ? currentCenturies.filter(value => value !== century)
+      : [...currentCenturies, century];
+  };
```

Comment:
The `handleToggleCentury` function returns an array of strings, but the `params` prop in the `SearchLink` component expects an object. Ensure that the `params` prop is correctly structured to handle an array of centuries. Consider using `getSearchWith` or a similar utility function to properly format the search parameters.